packages/design/src/colors.css

```diff
@@ -17,24 +17,24 @@
   /* Status Colours */
 
   --color-critical: var(--color-red);
-  --color-critical--light: var(--color-red--lightest);
-  --color-critical--dark: var(--color-red--dark);
+  --color-critical--surface: var(--color-red--lightest);
+  --color-critical--onSurface: var(--color-red--dark);
 
   --color-warning: var(--color-yellow);
-  --color-warning--light: var(--color-yellow--lightest);
-  --color-warning--dark: var(--color-yellow--dark);
+  --color-warning--surface: var(--color-yellow--lightest);
+  --color-warning--onSurface: var(--color-yellow--dark);
 
   --color-success: var(--color-green);
-  --color-success--light: var(--color-green--lightest);
-  --color-success--dark: var(--color-green--dark);
+  --color-success--surface: var(--color-green--lightest);
+  --color-success--onSurface: var(--color-green--dark);
 
   --color-informative: var(--color-lightBlue);
-  --color-informative--light: var(--color-lightBlue--lightest);
-  --color-informative--dark: var(--color-lightBlue--dark);
+  --color-informative--surface: var(--color-lightBlue--lightest);
+  --color-informative--onSurface: var(--color-lightBlue--dark);
 
   --color-inactive: var(--color-blue--light);
-  --color-inactive--light: var(--color-blue--lightest);
-  --color-inactive--dark: var(--color-blue--dark);
+  --color-inactive--surface: var(--color-blue--lightest);
+  --color-inactive--onSurface: var(--color-blue--dark);
```

Comment:
Renaming these status variants to be more clear about their intended purpose

Reply:
So if i understand these correct, If i had a component with a background of `--color-success--surface` anything on top of that would use `--color-success--onSurface`

Reply:
Yep!

Reply:
This is not entirely reflective of all current elements that could use this approach (ie Banner), but holds true for elements like InlineLabel, and I think we can look at making Banner, Toast, etc use this approach more consistently.